index.rb

```diff
@@ -0,0 +1,19 @@
+def morse
+  { A: '.-', B: '-...', C: '-.-.', D: '-..', E: '.', F: '..-.', G: '--.', H: '....',
+    I: '..', J: '.---', K: '-.-', L: '.-..', M: '--', N: '-.', O: '---', P: '.--.',
+    Q: '--.-', R: '.-.', S: '...', T: '-', U: '..-', V: '...-', W: '.--', X: '-..-',
+    Y: '-.--', Z: '--..', '/': '/' }
+end
+
+def decode_char(char)
+  morse.key char
+end
```

Comment:
- [ ] According to the project requirements, when you run `p decode_char('.-')` against your code, the output should be `"A"` but yours returned `:A`. Kindly fix it.

![image](https://user-images.githubusercontent.com/16058470/135653341-e8d432ae-5390-40bb-9a77-ce781f4bce03.png)

Reply:
Hello @ElijahTobs you have to use **_puts_** instead of _**p**_.  If you use  **p** --> it prints everything. For Example if you use it for sting it will show you **NOT** only the text BUT the quotes as well, If you use it for array it will show you the brackets as well:

Here is example, you can test it in you irb as well with different examples. Since we are using hashes p will print :A not A

![image](https://user-images.githubusercontent.com/79217155/135657580-04fa036b-40ff-4c4e-84c7-f3040f629fba.png)

![image](https://user-images.githubusercontent.com/79217155/135657635-cf169d8b-f7e4-4c10-a8c0-573eabb6fc4f.png)


Please use puts and check the morse code again! 

Here https://flexiple.com/puts-vs-p-vs-print-ruby/ you can check it or you can find some of yourself.

Reply:
![image](https://user-images.githubusercontent.com/79217155/135659151-c97dc051-4403-47c7-a500-3ad80e291470.png)